frontend/components.d.ts

```diff
@@ -39,6 +39,7 @@ declare module '@vue/runtime-core' {
     Loading: typeof import('./src/components/common/Loading.vue')['default']
```

Comment:
Diese Datei muss nicht commited werden. Der Eintrag befindet sich in der gitignore des Wurzelverzeichnisses.

![image](https://github.com/it-at-m/isi-frontend/assets/56300801/479276e0-9e08-4efc-acf8-9d6319b03b58)